pynetbox/models/ipam.py

```diff
@@ -97,3 +97,38 @@ def available_prefixes(self):
 class Aggregates(Record):
     def __str__(self):
         return str(self.prefix)
+
+
+class Vlans(Record):
+    def __str__(self):
+        return (
+            getattr(self, "display", None)
+            or getattr(self, "name", None)
+            or str(self.vid)
+        )
```

Comment:
Using Record's str method might save us from having to remember this if another similar field is added. E.g.
```suggestion
        return super().__str__(self) or str(self.vid)
```
Can see the benefits of keeping it this way too. What do you think?

Reply:
Never thought about it that way, looks good!